pkg/openstackipset/configmap.go

```diff
@@ -24,13 +24,14 @@ import (
 	"strings"
 
 	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	ospdirectorv1beta1 "github.com/openstack-k8s-operators/osp-director-operator/api/v1beta1"
 	common "github.com/openstack-k8s-operators/osp-director-operator/pkg/common"
 	"k8s.io/apimachinery/pkg/types"
 )
```

Comment:
I think the changes to this file would be worth going over in detail in the call tomorrow